web/demos/kachaka_api_web_sample/src/App.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback, useState } from 'react'
+
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import CssBaseline from '@mui/material/CssBaseline'
+import Box from '@mui/material/Box'
+import Container from '@mui/material/Container'
+import Fab from '@mui/material/Fab'
+import Grid from '@mui/material/Grid'
+import MuiAppBar from '@mui/material/AppBar'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import AddIcon from '@mui/icons-material/Add'
+
+import { v4 as uuidV4 } from 'uuid'
+import { GrpcWebFetchTransport } from '@protobuf-ts/grpcweb-transport'
+
+import { KachakaApiClient } from './protos/kachaka-api.client'
+import { KachakaApiPanel } from './KachakaApiPanel'
+import { useLocations, useShelves } from './kachakaApi'
+
+const defaultTheme = createTheme()
+
+function App(): JSX.Element {
+  const kachakaApiClient = new KachakaApiClient(
+    new GrpcWebFetchTransport({
+      baseUrl: 'http://localhost:50000',
+    }),
+  )
+  const locations = useLocations(kachakaApiClient)
+  const shelves = useShelves(kachakaApiClient)
+
+  const [panelIds, setPanelIds] = useState<string[]>([uuidV4()])
```

Comment:
この書き方だと、App() の実行の度に uuidV4() が実行されてしまうので、useState には関数を渡すようにするとよい、だったはず。
```suggestion
  const [panelIds, setPanelIds] = useState<string[]>(() => [uuidV4()])
```